packages/devtools-core/tests/components/DevTools/components/JsonTree.test.tsx

```diff
@@ -0,0 +1,18 @@
+import "@babel/polyfill";
+import * as React from "react";
+import JsonTree from "../../../../src/App/DevTools/components/JsonTree";
+import { model } from "../../../../src/model";
+import { initState } from "../../../../src/store";
+import { renderWithProdo } from "../../../utils";
+
+describe("JsonTree", () => {
+  it("renders the value", async () => {
+    const value = { foo: "bar" };
+    const { getByTestId } = renderWithProdo(
```

Comment:
Do we need `renderWithProdo` here? Not sure that the model is used at all

Reply:
The model isn't, but the ThemeProvider is (styled components use it for some props). Do you think it's worth making a separate renderWithTheme function that only applies and theme, not the model? It'd only be used here at the moment, but I guess later in other components too. Or I can just use renderwithProdo for everything.